2025-04-25 00:12:12,126 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:14:24,420 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:14:24,741 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:20:45,035 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:22:20,417 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:110]
2025-04-25 00:22:53,534 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:23:55,187 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:24:28,264 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:24:28,582 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:24:32,643 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-04-25 00:40:32,720 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-04-25 11:43:02,235 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 11:43:02,669 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 01:20:07,681 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 01:20:07,904 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 01:20:12,064 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-05-02 01:20:12,066 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-05-02 01:20:12,144 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-05-02 01:39:46,658 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 01:39:46,858 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:03:38,951 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:03:39,156 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:04:40,520 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:06:15,210 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:07:26,424 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:11:38,408 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:16:48,259 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-05-02 02:43:43,437 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:43:43,685 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:44:18,355 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:48:19,652 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:48:19,872 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:48:20,131 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:53:28,162 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:53:28,382 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:53:28,624 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 02:53:52,461 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 03:12:27,421 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 03:12:27,623 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 15:05:13,389 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-05-02 15:42:17,829 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-05-02 15:47:29,691 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-05-02 15:55:03,218 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:81]
2025-05-02 15:55:03,444 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:81]
2025-05-02 15:56:38,944 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:90]
2025-05-02 15:57:04,348 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:90]
